### PR TITLE
2808 - added imports/exports to Slider

### PR DESCRIPTION
**Pull request by eleanor-heath**
ES6 code conversion was completed in https://github.com/LearningPool/adapt-contrib-slider/pull/3

The purpose of this PR is to add in the imports/exports to the component.

https://github.com/adaptlearning/adapt_framework/issues/2808

### CODE REVIEW

**Review by ethan-lp — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: js/sliderView.js [hunk 1]
```diff
@@ -1,363 +1,358 @@
...
-      });
-      this.oldValue = 0;
+    this.selectItem(this.getIndexFromValue(this.model.get('_selectedItem').value));
+  }
 
-      if (this._deferEnable) {
-        this.setAllItemsEnabled();
-      }
+  setupRangeslider() {
+    this.rangeslider = rangeslider;
```
Comment (oliverfoster) [inline js/sliderView.js:23]: weird addition, intentional?

Comment (cahirodoherty-learningpool) [inline js/sliderView.js:23]: I don't think this line is needed

File: js/sliderView.js [hunk 2]
```diff
@@ -1,363 +1,358 @@
...
-
-    // Used by question to setup itself just before rendering
-    setupQuestion() {
-      if (this.model.get('_isSubmitted')) return;
-
-      this.selectItem(this.getIndexFromValue(this.model.get('_selectedItem').value));
-    }
+import Adapt from 'core/js/adapt';
+import QuestionView from 'core/js/views/questionView';
+import rangeslider from 'libraries/rangeslider';
```
Comment (oliverfoster) [inline js/sliderView.js:3]:
```suggestion
import 'libraries/rangeslider';
```
this should work

File: js/sliderView.js (hunk 1 above)
Comment (oliverfoster) [reply js/sliderView.js:23]: i get it, you had to declare and use the variable from the import?

**Review by oliverfoster — CHANGES_REQUESTED** [review]

File: js/sliderView.js (hunk 1 above)
Comment (eleanor-heath) [reply js/sliderView.js:23]: thats why I added it in yes

File: js/sliderView.js (hunk 2 above)
Comment (eleanor-heath) [reply js/sliderView.js:3]: gotcha thanks

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**